homeassistant/components/zha/api.py

```diff
@@ -110,6 +119,26 @@
     vol.Required(ATTR_CLUSTER_TYPE): str
 })
 
+WS_BIND_DEVICE = 'zha/devices/bind'
```

Comment:
Please use the new websocket registration method

https://github.com/home-assistant/home-assistant/blob/8a026bf2146e7196bd26db03736fd22ffc387c26/homeassistant/components/system_health/__init__.py#L56-L59

Reply:
https://github.com/home-assistant/home-assistant/pull/21533 i will rebase this draft PR and convert that there if that is ok